src/pages/dmt5/_dtrader-video.js

```diff
@@ -0,0 +1,282 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { localize } from 'components/localization'
+import { Header } from 'components/elements/typography.js'
+// import MacBook from 'images/svg/macbook.svg'
+import Image from 'components/elements/image'
+import device from 'themes/device.js'
+
+const Container = styled.section`
+    width: 100%;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-rows: auto;
+    grid-column-gap: 2rem;
+    grid-template-areas:
+        'tabs'
+        'video';
+
+    @media ${device.tabletL} {
+        grid-template-areas:
+            'video'
+            'tabs';
+    }
+`
+const TabsWrapper = styled.div`
+    grid-area: tabs;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    margin-top: 4rem;
+
+    @media ${device.tabletL} {
+        flex-direction: column;
+        justify-content: flex-start;
+    }
+`
+const StepCommon = css`
+    font-weight: 500;
+    cursor: pointer;
+    @media ${device.tabletL} {
+        text-align: left;
+        border: none;
+        margin-top: ${props => (props.no_margin ? '0' : '2rem')};
+    }
+`
+const Step1 = styled(Header)`
+    ${StepCommon}
+    ${props =>
+        props.current_step_image === 'dmt-5-step-1.png'
+            ? 'color: var(--color-red); border-bottom: 1px solid var(--color-red)'
+            : 'color: var(--color-red-2); border-bottom: 1px solid var(--color-red-2)'};
+`
+const Step2 = styled(Header)`
+    ${StepCommon}
+    ${props =>
+        props.current_step_image === 'dmt-5-step-2.png'
+            ? 'color: var(--color-red); border-bottom: 1px solid var(--color-red)'
+            : 'color: var(--color-red-2); border-bottom: 1px solid var(--color-red-2)'};
+`
+const Step3 = styled(Header)`
+    ${StepCommon}
+    ${props =>
+        props.current_step_image === 'dmt-5-step-3.png'
+            ? 'color: var(--color-red); border-bottom: 1px solid var(--color-red)'
+            : 'color: var(--color-red-2); border-bottom: 1px solid var(--color-red-2)'};
+`
+const Tab = styled.div`
+    width: 29rem;
+`
+const VideoWrapper = styled.div`
+    grid-area: video;
+    margin-top: 4rem;
+    position: relative;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+`
+// const MacbookFrame = styled(MacBook)`
```

Comment:
Why all this commented out code? if it's not needed please remove it as we can always retrieve it from our source control if we need it in the future.